release-team/README.md

```diff
@@ -185,3 +185,26 @@ is undefined.
 ### During "Security" releases
 For all unplanned or embargoed releases
 - Facilitate security releases following the under the [Security Release Process](security-release-process-documentation/security-release-process.md)
+
+---
+
+## Milestone Maintainers
+
+Across release cycles, one of the best signals for [issue triage](https://github.com/kubernetes/community/blob/master/contributors/guide/issue-triage.md#milestones) is whether or not an issue or PR is actually targeted for the current milestone.
+
+To maintain up-to-date status on milestone inclusion, we rely on a set of Milestone Maintainers (members of the `kubernetes-milestone-maintainers` GitHub team) to apply the appropriate labels to issues / PRs. This is facilitated by `/milestone` commands and bot automation.
+
+Each release cycle, the current Release Team Lead must update membership to the aforementioned GitHub team.
+
+Membership to the `kubernetes-milestone-maintainers` GitHub team is defined as follows:
+- Maintainers:
+  - SIG Release Chairs
+  - Current Release Team Lead
+- Members
+  - (at least 1, up to 2) Member(s) of SIG leadership (SIG Chairs / Technical Leads) from all SIGs
+  - (at least 1, up to 2) SIG milestone maintainers from all SIGs (in addition to SIG leadership)
+  - Special code reviewers, as selected by SIG Release
+
```

Comment:
It's important for us to have membership criteria here documented, especially as we will soon move to this team being managed via a yaml file in the k/k repo and thus PRs against it should have a clear review criteria.

But I still think this is overly specific.  I'd like to see this more pragmatic and less proscriptive on the broader k8s community.  If a SIG does not and has not had anybody on the list, that's not a problem in my mind.  It means they've:
 *  had all features captured early in conjunction with the Features Lead from SIG Release who set the milestone (or whatever that process evolves to between SIG Release and SIG PM).
 *  had all required code merged ahead of code freeze
That's a well performing SIG.  It adds process on them to require them keep their milestone maintainers membership up to date for no utility.

What do you think of this wording, with the word "should" being deliberate versus "must" or "at least N, up to M", because I believe it's not just okay but actually normal case good if a SIG is getting by with zero members:

Membership to the `kubernetes-milestone-maintainers` GitHub team should comprise:
  - SIG Release Chairs 
  - Current Release Team Lead
  - Per-SIG leadership (SIG Chairs / Technical Leads / Approver delegates)
  - Special code reviewers, as selected by SIG Release for their exceptional contribution activity and project oversight